CMakeLists.txt

```diff
@@ -11,17 +11,9 @@ endif()
 FetchContent_Declare(
   mlx-c
   GIT_REPOSITORY "https://github.com/ml-explore/mlx-c.git"
-  GIT_TAG "v0.0.8")
+  GIT_TAG "v0.0.9")
 FetchContent_MakeAvailable(mlx-c)
 
-# TEMPORARY OVERRIDE -- 0.0.8 depends on v0.14.0 but we need v0.15.2 for iOS /
-# float16 issues
```

Comment:
I may have to add this back as mlx-c is for v0.15.2 but we need to build vs v0.16.0 for the Sequoia assert crashes